plugins/commands/create.js

```diff
@@ -97,16 +97,11 @@ module.exports = () => ({
 
             // if Content-type is multipart/form-data, both command binary and meta are posted
             if (request.headers['content-type'].startsWith('multipart/form-data')) {
-                if (data.file === undefined || data.file.length !== 2) {
+                if (data.binary === undefined || data.spec === undefined) {
```

Comment:
According to the sd-cmd PR comment (https://github.com/screwdriver-cd/sd-cmd/pull/13#issuecomment-371022838), sd-cmd cli POST a multipart/form-data which includes spec part and binary part. Therefore, I fixed commands API depending on this change.